src/app/containers/StoryPromo/MediaIndicator/index.test.jsx

```diff
@@ -21,6 +39,16 @@ const item = {
   },
 };
 
+const photogalleryItem = {
+  headlines: {
+    headline: 'A photo gallery item',
+  },
+  locators: {
+    assetUri: 'https://www.bbc.co.uk',
+  },
+  cpsType: 'PGL',
```

Comment:
Why are we not including `media.format` for `PGL`?

Reply:
The PGL `cpsType` doesn't have a media object in the feed. 
https://github.com/bbc/simorgh/blob/latest/data/pidgin/frontpage/index.json#L4868